File: services/catalog-service/src/main/java/com/tap2eat/catalog/dtos/response/internal/ValidatedModifierResponse.java
package com.tap2eat.catalog.dtos.response.internal;

import java.math.BigDecimal;

public record ValidatedModifierResponse(
        String modifierGroupId,
        String modifierGroupName,
        String modifierOptionId,
        String modifierOptionName,
        BigDecimal priceAdjustment
) {
}

File: services/order-service/Integrations/Catalog/Dtos/ValidatedModifierResponse.cs
namespace OrderService.Integrations.Catalog.Dtos;

public sealed class ValidatedModifierResponse
{
    public string? ModifierGroupId { get; set; }

    public string ModifierGroupName { get; set; } = string.Empty;

    public string? ModifierOptionId { get; set; }

    public string ModifierOptionName { get; set; } = string.Empty;

    public decimal PriceAdjustment { get; set; }
}
